src/WebJobs.Extensions.DurableTask/DurableOrchestrationClient.cs

```diff
@@ -42,6 +46,38 @@ public class DurableOrchestrationClient : DurableOrchestrationClientBase
             this.attribute = attribute;
         }
 
+        /// <summary>
+        /// Constructor for <see cref="DurableOrchestrationClient"/>
+        /// </summary>
+        /// <param name="connectionName">Azure Storage connection name.</param>
+        /// <param name="taskHubName">Name of task hub.</param>
+        public DurableOrchestrationClient(string connectionName, string taskHubName)
+        {
+            this.config = new DurableTaskExtension();
+            JobHostConfiguration jobHostConfiguration =
+                new JobHostConfiguration(connectionName)
+                {
+                    LoggerFactory = new LoggerFactory(),
+                };
+            ExtensionConfigContext extensionConfigContext = new ExtensionConfigContext
+            {
+                Config = new JobHostConfiguration(connectionName),
```

Comment:
You've already created a `JobHostConfiguration` a few lines above this. Why not use that one instead of creating another?

Reply:
Thank you! I will fix it.